Sources/Basics/Concurrency/AsyncProcess.swift

```diff
@@ -501,7 +501,7 @@ package final class AsyncProcess {
 
             group.enter()
             stdoutPipe.fileHandleForReading.readabilityHandler = { (fh: FileHandle) in
-                let data = fh.availableData
+                let data = fh.readData(ofLength: Int.max)
```

Comment:
This method is deprecated, `read(upToCount:)` is the recommended replacement.

Reply:
Good suggestion. Thank you!